### PR TITLE
i18n(es): Update `plugins.md`, `configuration.mdx` & `showcase.mdx`

### PR DESCRIPTION
**Pull request by Waxer59**
#### Description

- What does this PR change? Give us a brief description.
Update `plugins.md`, `configuration.mdx` & `showcase.mdx` [`87659f0`](https://github.com/withastro/starlight/commit/87659f061ea6d73be4bd48063a2e3f025adb0514)

### CODE REVIEW

File: docs/src/content/docs/es/showcase.mdx [hunk 1]
```diff
@@ -18,11 +18,36 @@ Starlight ya está siendo utilizado en producción. Estos son algunos de los sit
...
+Los [Plugins](/es/reference/plugins) pueden personalizar la configuración, UI y el comportamiento de Starlight, al mismo tiempo que son fáciles de compartir y reutilizar.
+Extiende tu sitio con plugins oficiales respaldados por el equipo de Starlight y plugins de la comunidad mantenidos por usuarios de Starlight.
+
+### Plugins oficiales
+
+<CardGrid>
+	<LinkCard
+		href="/guides/site-search/#algolia-docsearch"
+		title="Algolia DocSearch"
+		description="Reemplaza Pagefind, el proveedor de búsqueda predeterminado, con Algolia DocSearch."
```
Comment (dreyfus92) [inline docs/src/content/docs/es/showcase.mdx:32]: Te dejo una pequeña sugerencias @Waxer59, buen trabajo 😁
```suggestion
		description="Reemplaza Pagefind, el proveedor de búsqueda predeterminado con Algolia DocSearch."
```

**Comment by Waxer59** [comment]
Listo!, muchas gracias por la revisión 🤠

**Review by dreyfus92 — APPROVED** [review]
LGTM ✅

File: docs/src/content/docs/es/showcase.mdx [hunk 2]
```diff
@@ -18,11 +18,36 @@ Starlight ya está siendo utilizado en producción. Estos son algunos de los sit
...
+## Plugins
+
+Los [Plugins](/es/reference/plugins) pueden personalizar la configuración, UI y el comportamiento de Starlight, al mismo tiempo que son fáciles de compartir y reutilizar.
+Extiende tu sitio con plugins oficiales respaldados por el equipo de Starlight y plugins de la comunidad mantenidos por usuarios de Starlight.
+
+### Plugins oficiales
+
+<CardGrid>
+	<LinkCard
+		href="/guides/site-search/#algolia-docsearch"
```
Comment (delucis) [inline docs/src/content/docs/es/showcase.mdx:30]: One detail here, this link should be to the Spanish page:

```suggestion
		href="/es/guides/site-search/#algolia-docsearch"
```

Easy to miss with the format so different from links.

**Review by delucis — CHANGES_REQUESTED** [review]
Thanks for the PR @Waxer59! I noticed one small link issue.

Could you also update this PR with the changes from #1301? We moved one more plugin.

**Comment by Waxer59** [comment]
review and new changes applied, thanks for the review 🤠!

File: docs/src/content/docs/es/showcase.mdx [hunk 3]
```diff
@@ -45,6 +46,25 @@ Estas herramientas de la comunidad, plugins e integraciones funcionan junto a St
 		title="starlight-typedoc"
 		description="Genera páginas de Starlight a partir de TypeScript usando TypeDoc."
 	/>
+</CardGrid>
+
+## Herramientas e integraciones de la comunidad
+
+import { CardGrid, LinkCard } from '@astrojs/starlight/components';
+
+Estas herramientas de la comunidad e integraciones pueden utilizarse para añadir características a tu sitio de Starlight.
```
Comment (kevinzunigacuellar) [inline docs/src/content/docs/es/showcase.mdx:55]:
```suggestion
Estas herramientas de la comunidad e integraciones pueden utilizarse para añadir funcionalidades a tu sitio de Starlight.
```

File: docs/src/content/docs/es/reference/plugins.md [hunk 4]
```diff
@@ -8,7 +8,7 @@ tableOfContents:
 Los plugins de Starlight pueden personalizar la configuración, la UI y el comportamiento, mientras que son fáciles de compartir y reutilizar.
 Esta página de referencia documenta la API a la que tienen acceso los plugins.
 
-Aprende más sobre el uso de un plugin de Starlight en la [Referencia de Configuración](/es/reference/configuration/#plugins).
+Aprende más sobre el uso de un plugin de Starlight en la [Referencia de Configuración](/es/reference/configuration/#plugins) o visita el [mostrador de plugins](/es/showcase/#plugins) para ver una lista de los plugins disponibles.
```
Comment (kevinzunigacuellar) [inline docs/src/content/docs/es/reference/plugins.md:11]:
```suggestion
Aprende más sobre el uso de un plugin de Starlight en la [Referencia de Configuración](/es/reference/configuration/#plugins) o visita la [exhibición de plugins](/es/showcase/#plugins) para ver una lista de los plugins disponibles.
```

File: docs/src/content/docs/es/showcase.mdx [hunk 5]
```diff
@@ -4,7 +4,7 @@ description: ¡Descubre los sitios construidos con Starlight y herramientas comu
 ---
 
 :::tip[¡Agrega el tuyo!]
-¿Has construido un sitio de Starlight o una herramienta para Starlight?
+¿Has construido un sitio de Starlight, plugin o una herramienta para Starlight?
```
Comment (kevinzunigacuellar) [inline docs/src/content/docs/es/showcase.mdx:7]:
```suggestion
¿Has construido un sitio de Starlight, un plugin o una herramienta para Starlight?
```

**Review by kevinzunigacuellar — COMMENTED** [review]
Gracias por la PR @Waxer59 y la revisión @dreyfus92 . Dejé unos pequeños comentarios

**Comment by Waxer59** [comment]
Cambios aplicados, muchas gracias por la revisión @kevinzunigacuellar 🤠!

**Review by kevinzunigacuellar — APPROVED** [review]

**Comment by dreyfus92** [comment]
Is there something still missing on this PR? cc @delucis @kevinzunigacuellar

**Review by delucis — APPROVED** [review]
Looks great! Thanks for your patience over the holidays 🤗

**Pull request merged**